### PR TITLE
feat: added github links to footer

### LINKED ISSUE #11: Build a dummy footer
Something like:

https://tailwindcomponents.com/component/responsive-footer-2

Screenshot 2021-03-31 at 00 10 04

### PR DESCRIPTION
**Pull request by samnaish**
added new Github links,
made footer more responsive ( it grows with content ).

ToDo:
add more styling ie hover

Closes #11

### CODE REVIEW

**Review by alexnaish — COMMENTED** [review]
Few minor comments, also - it looks like the preview deploy URLs only support lower case branch names (footer not Footer). I'll look into that separately though.

**Comment by alexnaish** [comment]
Also, lets try to put a little more content into it - check the details on the issue (https://github.com/AlexNaishLtd/management-portal/issues/11)

File: components/Footer.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
-}
+    return (
+        <footer className="text-center sm:mt-10 h-auto bg-blue-500">
+            <span className="m-8">A project developed by Alex and Sam Naish</span>
+            <div className="flex flex-wrap flex-col">
+                <h4 className="pt-2">Community Links</h4>
+            </div>
+            <div className="flex flex-row justify-center">
+                <div className="py-2 px-2">
+                    <Links linkRef="https://github.com/alexnaish" label="Alex Naish Github" />
```
[the remark this answers is not in the document]
Comment (samnaish) [reply components/Footer.js:12]: I initially tried to name it but Link module that next has conflicts with the name of that component.
So If I named it "Alink" would that be just as good?

[the remark this answers is not in the document]
Comment (alexnaish) [reply components/Footer.js]: Ah good point, in this case then, since it'll be styled to fit in with the Footer component - lets call it a `FooterLink`

**Review by alexnaish — APPROVED** [review]

**Pull request merged**